client/my-sites/plan-features/index.jsx

```diff
@@ -88,27 +93,31 @@ PlanFeatures.propTypes = {
 	available: PropTypes.bool,
 	onUpgradeClick: PropTypes.func,
 	// either you specify the plan prop or isPlaceholder prop
-	plan: React.PropTypes.string,
-	isPlaceholder: PropTypes.bool
+	plan: PropTypes.string,
+	isPlaceholder: PropTypes.bool,
+	isInSignup: PropTypes.bool
```

Comment:
Sorry, I don't follow you on this one. :) Call `isInSignup` `useSiteSpecificPlans`? If yes, how is that related? What's site specific plans? The `/sites/$siteId/plans` API result?